r2-core/src/main/java/com/linkedin/r2/transport/http/client/SmoothRateLimiter.java

```diff
@@ -225,6 +224,7 @@ public int getPendingTasksCount(){
     private int _permitAvailableCount;
     private int _permitsInTimeFrame;
     private long _nextScheduled;
+    private boolean _wasDelayed;
```

Comment:
Shall we initialize this to `false` in the constructor?

Reply:
Same same. I can explicitly set to false in the constructor if that's the preferred style.